src/test/java/io/rtr/conduit/amqp/impl/AMQPAsyncConsumerBuilderTest.java

```diff
@@ -39,6 +40,53 @@ public void testValidationQueueRequiredWhenNotDynamic(){
     }
 
     @Test
+    public void testValidationBasicConfig(){
+        AMQPAsyncConsumerBuilder amqpAsyncConsumerBuilder = AMQPAsyncConsumerBuilder.builder()
+                .autoCreateAndBind("exchange", AMQPConsumerBuilder.ExchangeType.DIRECT, "queue", "routingKey");
+        AMQPCommonListenProperties commonListenProperties = amqpAsyncConsumerBuilder.buildListenProperties();
+
+        assertEquals("Can not autoCreateAndBind and be dynamic", false, commonListenProperties.isDynamicQueueCreation());
```

Comment:
my point is its not like we're trying to induce the bad combination and verifying that it can't happen we're simply asserting that the dynamic queue property is keeping its default value.  if the intent is to safeguard against changes to the default value of the dynamic queue property then state that and perhaps invoke validate so that the validation is exercised.